lib/widgets/icons.dart

```diff
@@ -78,53 +78,56 @@ abstract final class ZulipIcons {
   /// The Zulip custom icon "inbox".
   static const IconData inbox = IconData(0xf112, fontFamily: "Zulip Icons");
 
+  /// The Zulip custom icon "info".
+  static const IconData info = IconData(0xf113, fontFamily: "Zulip Icons");
```

Comment:
nit in commit message:

    The icon is taken from Zulip main repo:
    https://github.com/zulip/zulip/blob/74d7f1192/web/shared/icons/info.svg

"Zulip main repo" -> "Zulip web"

(I'd usually call the repo the Zulip server/web repo; here we can be more specific as it's from the web client and not the server)